From 83d89c383f323784e99c4330866c80f0054ba0ff Mon Sep 17 00:00:00 2001
From: Olivier Brunel <jjk@jjacky.com>
Date: Thu, 17 Oct 2013 20:19:21 +0200
Subject: [PATCH] Fix key "Page_Down" identified as "Next"

Since 49c4ad2f the order between "Page_Down" & "Next" had been changed (due to
sorting) and this resulted in the Page Down key being identified as "Next"
instead of "Page_Down"

"Page_Up" wasn't affected, since it comes before "Prior"

Signed-off-by: Olivier Brunel <jjk@jjacky.com>
---
 gdk/keyname-table.h | 10 +++++-----
 gdk/keynames.txt    |  2 +-
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/gdk/keyname-table.h b/gdk/keyname-table.h
index f921ea0..0df5498 100644
--- a/gdk/keyname-table.h
+++ b/gdk/keyname-table.h
@@ -1,6 +1,6 @@
 /* keyname-table.h: Generated by gen-keyname-table.pl from keynames.txt
  *
- *  Date: Wed Jul 17 21:51:09 2013
+ *  Date: Thu Oct 17 18:16:31 2013
  *
  * Do not edit.   
  */
@@ -1237,8 +1237,8 @@ static const char keynames[] =
   "Down\0"
   "Page_Up\0"
   "Prior\0"
-  "Next\0"
   "Page_Down\0"
+  "Next\0"
   "End\0"
   "Begin\0"
   "Select\0"
@@ -3515,7 +3515,7 @@ static const gdk_key gdk_keys_by_keyval[] = {
   { 0x00ff55, 14268 },
   { 0x00ff55, 14276 },
   { 0x00ff56, 14282 },
-  { 0x00ff56, 14287 },
+  { 0x00ff56, 14292 },
   { 0x00ff57, 14297 },
   { 0x00ff58, 14301 },
   { 0x00ff60, 14307 },
@@ -5488,7 +5488,7 @@ static const gdk_key gdk_keys_by_name[] = {
   { 0x1008ff68, 26441 },
   { 0x10020aa, 20596 },
   { 0x1008ff69, 26445 },
-  { 0x00ff56, 14282 },
+  { 0x00ff56, 14292 },
   { 0x1008fe22, 25553 },
   { 0x00fed2, 13117 },
   { 0x0000d1, 881 },
@@ -5527,7 +5527,7 @@ static const gdk_key gdk_keys_by_name[] = {
   { 0x00fe79, 12903 },
   { 0x000050, 237 },
   { 0x1001e56, 19027 },
-  { 0x00ff56, 14287 },
+  { 0x00ff56, 14282 },
   { 0x00ff55, 14268 },
   { 0x1008ff6d, 26473 },
   { 0x00ff13, 13705 },
diff --git a/gdk/keynames.txt b/gdk/keynames.txt
index 18ed271..92f1770 100644
--- a/gdk/keynames.txt
+++ b/gdk/keynames.txt
@@ -1230,8 +1230,8 @@
 0x00ff54 Down
 0x00ff55 Page_Up
 0x00ff55 Prior
-0x00ff56 Next
 0x00ff56 Page_Down
+0x00ff56 Next
 0x00ff57 End
 0x00ff58 Begin
 0x00ff60 Select
-- 
1.8.4.1

